src/plugins/email-smtp/util/getMailConfig.js

```diff
@@ -119,21 +53,7 @@ export default async function getMailConfig(context, shopId) {
   if (MAIL_URL) return getConfigFromMailUrl({ logger, mailUrl: MAIL_URL });
 
   /**
-   * Fall back to a standard service
-   */
-  const settings = await Packages.findOne({ name: "core", shopId }) || {};
-  const shopSettings = settings.settings || {};
-  const { service, user, password, host, port } = shopSettings.mail || {};
-
-  if (service && service !== "custom") return getStandardServiceConfig({ logger, password, service, user });
-
-  /**
-   * Fall back to a custom service
-   */
-  if (host && port) return getCustomServiceConfig({ host, logger, password, port, user });
-
-  /**
-   * If all else fails, try direct sending but warn that this rarely works.
+   * Try direct sending but warn that this rarely works.
```

Comment:
In the other function, we just fail if `direct` is true, so this message isn't accurate. I think the best would be:

- Remove all of this after `if (MAIL_URL)` line and just throw an error instead.
- In `sendSMTPEmail.js`, add `try/catch` around `getMailConfig()` line, and call `sendEmailFailed` in the `catch` block. No need to check `config.direct` in that file.

Reply:
suggestions implemented.